src/shared/claim-page.tsx

```diff
@@ -35,14 +42,10 @@ export const ClaimPage: React.FC<React.PropsWithChildren<ClaimPageProps>> = ({
             placeContent="center"
             overflow="hidden"
           >
-            {metadata?.image ? (
-              <Image
-                objectFit="contain"
-                w="100%"
-                h="100%"
-                src={metadata?.image}
-                alt={metadata?.name}
-              />
+            {tokenId && nft?.metadata ? (
+              <ThirdwebNftMedia metadata={nft.metadata} />
+            ) : metadata?.image ? (
+              <ThirdwebNftMedia metadata={metadata} />
```

Comment:
yeah ideally we split these high level into 2 components, one for erc20 and erc721 and one for erc1155